contracts/ROSCA.sol

```diff
@@ -611,9 +611,20 @@ contract ROSCA {
     selfdestruct(foreperson);
   }
 
-	////////////////////
-	// HELPER FUNCTIONS
-	////////////////////
+  ////////////////////
+  // HELPER FUNCTIONS
+  ////////////////////
+
+
+  /**
+   * @dev calculates the user's discount amount from the total discount
+   * @return uint256
+   */
+  function userTotalCredit(address memberAddress) internal constant returns (uint256) {
+    uint256 userDiscount = totalDiscounts / membersAddresses.length;
+
+  	return SafeMath.add(members[memberAddress].credit, userDiscount);
```

Comment:
Why's there an extra indentation here?

Reply:
should be fixed now, it was caused by my IDE